e2e/src/main/protobuf/sealed_oneof_extends_universal.proto

```diff
@@ -0,0 +1,37 @@
+syntax = "proto3";
+
+package com.thesamet.proto.e2e.universal;
```

Comment:
What would be a good test that the generator does the right thing and it's not just happening to compile? This would be useful to prevent regressions if changes in the generator are introduced later.